spec/libnotify_spec.cr

```diff
@@ -45,3 +45,5 @@ describe Libnotify do
     end
   end
 end
+
+require "./notification_spec"
```

Comment:
I don't think you need to `require` the spec here. It will be automatically picked by `make spec`.